types/react-filepond/index.d.ts

```diff
@@ -15,39 +15,73 @@ import * as React from 'react';
 // exporting the API that react-filepond exports.
 export {};
 
+export enum FileStatus {
+    INIT = 1,
+    IDLE = 2,
+    PROCESSING_QUEUED = 9,
+    PROCESSING = 3,
+    PROCESSING_COMPLETE = 5,
+    PROCESSING_ERROR = 6,
+    PROCESSING_REVERT_ERROR = 10,
+    LOADING = 7,
+    LOAD_ERROR = 8,
+}
+
+export enum Status {
```

Comment:
Rather than updating this Definitely Typed package, I think it might be time to remove it.  

Since I published this package, filepond has released [its own typescript definitions](https://github.com/pqina/filepond/blob/master/types/index.d.ts).

Is there any types here that you believe aren't covered by those types?

Reply:
Their regular filepond type definitions are fine, but don't include any of the react-filepond types like the FilePond or File react components.
Since their TS classes share these names it would be difficult to combine that type definition file with the React component.
Do you think it would be worthwhile to rather add these files to their repo?

Reply:
@WillemDuPlessis I do think these type definitions would be better off in the React Filepond repo.  By putting them there you could leverage all of the types defined in the vanilla Filepond, instead of having to continue to duplicate them like you do when writing them in this repository.

Because the author recently (some time last year) added those type definitions to Filepond, I think that he would be receptive to having them added to his react repo as well

Reply:
Ah, I see that you've already created [that PR](https://github.com/pqina/react-filepond/pull/150), awesome!

Reply:
No problem. I'll close this PR and continue with the one into react-filepond itself. Thanks for all your help!